tests/Integration/ScriptsTest.php

```diff
@@ -219,110 +222,65 @@ public function test_enqueue_js_api_no_secret(): void {
 	 * @uses \Parsely\Parsely::update_metadata_endpoint
 	 * @uses \Parsely\Scripts::register_scripts
 	 * @uses \Parsely\Scripts::script_loader_tag
-	 * @group insert-js
+	 * @group enqueue-js
 	 */
 	public function test_enqueue_js_api_with_secret(): void {
-		ob_start();
-		$post_array = $this->create_test_post_array();
-		$post       = $this->factory->post->create( $post_array );
-		$this->go_to( '/?p=' . $post );
```

Comment:
Does this need a `$this->go_to_new_post();` replacement?

Reply:
I left it out because the only thing the API seems to need is an API secret key. Adding or removing a `$this->go_to_new_post();` doesn't seem to have any influence on the test and the functionality of `enqueue_js_api()`.